### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by PeterGillbr** [bellrichm/WeeWX-MQTTSubscribe#131] [Q&A] [answered]
Please forgive a potentially stupid newbie question but I can get the install instructions to work on my Raspberry PI environment.
Executing "wget -P $DOWNLOAD_ROOT https://github.com/bellrichm/WeeWX-MQTTSubscribe/archive/2.0.0.tar.gz" in the RPi terminal returns the error "wget: missing URL". What am I doing wrong? Thanks in advance for any help

**Comment by bellrichm** [comment, marked as answer]
I think you are missing a 'v' in front of the version indicator, **v**2.0.0.tar.gz
```
wget -P $DOWNLOAD_ROOT https://github.com/bellrichm/WeeWX-MQTTSubscribe/archive/v2.0.0.tar.gz
```

**Reply by PeterGillbr** [reply]
Thanks for the quick reply! Stupid mistake on my side. Now the installation works and the weewx.conf is as expected. Thanks again

**Reply by bellrichm** [reply]
Good to hear. It is a bit hard to see, at least for my old eyes. I’ll try to remember to add an example.
